ASP.NET Core_EF_CodeFirst/Middlewares/CustomExceptionMiddleware/ExceptionMiddleware.cs

```diff
@@ -0,0 +1,47 @@
+﻿using ASP.NET_Core_EF_CodeFirst.Models;
+using BLL.Models.Exceptions;
+using System.Net;
+
+namespace ASP.NET_Core_EF_CodeFirst.Middlewares.CustomExceptionMiddleware
+{
+    public class ExceptionMiddleware
+    {
+        private readonly RequestDelegate _next;
+        public ExceptionMiddleware(RequestDelegate next)
+        {
+            _next = next;
+        }
+        public async Task InvokeAsync(HttpContext httpContext)
+        {
+            try
+            {
+                await _next(httpContext);
+            }
+            catch(ValidationException ex)
+            {
+                await HandleExceptionAsync(httpContext, ex);
+            }
+            catch 
+            {
+                await HandleExceptionAsync(httpContext);
+            }
+        }
+
+        private Task HandleExceptionAsync(HttpContext httpContext, ValidationException ex)
```

Comment:
Please implement this method, which is very similar to the other overload but you should use validation exception message in response.